extensions/amp-a4a/0.1/refresh-manager.js

```diff
@@ -0,0 +1,341 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {isExperimentOn} from '../../../src/experiments';
+import {Services} from '../../../src/services';
+import {dev, user} from '../../../src/log';
+import {IntersectionObserverPolyfill} from '../../../src/intersection-observer-polyfill'; // eslint-disable-line max-len
+
+/**
+ * - continuousTimeMin: The amount of continuous time, in milliseconds, that
+ *   the creative must be on screen for in oreder to be considered "visible".
```

Comment:
Please see my [comment](https://github.com/ampproject/amphtml/pull/9535#discussion_r127852095). 
I didn't mean the order of the object last time. I meant there's a typo. `order` spelled as ` oreder` 😄 
Please revert the object order though (visiblePercentageMin comes first) ... to match the format in analytics... 
Sorry about the confusion.